protobuf/types/service.proto

```diff
@@ -60,6 +71,11 @@ message Service {
     repeated Parameter outputs = 7 [
       (gogoproto.moretags) = 'hash:"name:7" validate:"dive,required"'
     ];
+
+    // price for the task.
+    Price price = 9 [
+      (gogoproto.moretags) = 'hash:"name:9"'
```

Comment:
I think the `dive` is mandatory to tell the validator to check the struct.
And I guess this this a required parameter?
```suggestion
      (gogoproto.moretags) = 'hash:"name:9" validate:"dive,required"'
```

Reply:
After discussion, we set the prices to required and services will need to be redeployed